lib/ember-test-helpers/test-module.js

```diff
@@ -230,6 +230,13 @@ export default Klass.extend({
     this.container = items.container;
     this.registry = items.registry;
 
+
+    var viewRegistry = this.container.lookup('-view-registry:main');
+    if (viewRegistry) {
+      var View = Ember.__loader.require('ember-views/views/view').default;
+      View.views = viewRegistry;
```

Comment:
In all cases Ember should be using `-view-registry:main` then falling back to its internal `View.views`.  The `-view-registry:main` is setup during [Ember.Application.buildRegistry](https://github.com/emberjs/ember.js/blob/1d7b833664292bc6644adc41a7dd9a842ea078c5/packages/ember-application/lib/system/application.js#L889) (injection is setup [directly below](https://github.com/emberjs/ember.js/blob/1d7b833664292bc6644adc41a7dd9a842ea078c5/packages/ember-application/lib/system/application.js#L891)) which we are calling from [here](https://github.com/switchfly/ember-test-helpers/blob/3315c6de26b80d0520e05862df46954e82f2e886/lib/ember-test-helpers/build-registry.js#L44).

Where in Ember are we still using `View.views` without checking `-view-registry:main` first?